main/session/session_list.php

```diff
@@ -108,6 +110,137 @@
         header('Location: '.$url);
         exit;
         break;
+    case 'export_multiple':
+        $sessionList = explode(',', $idMultiple);
+        $tempZipFile = api_get_path(SYS_ARCHIVE_PATH).api_get_unique_id().'.zip';
+        $zip = new PclZip($tempZipFile);
+        $csvList = [];
+
+        foreach ($sessionList as $session_id) {
```

Comment:
Use `$sessionItemId`, then define `$session_id = $sessionItemId;` right before the call to `get_user_data()` (which uses `$session_id` as a global variable)